gst/nnstreamer/nnstreamer_conf.c

```diff
@@ -345,3 +353,36 @@ nnsconf_get_fullpath (const gchar * subpluginname, nnsconf_type type)
   g_free (filename);
   return ret;
 }
+
+/** @brief Public function defined in the header */
+const gchar *
+nnsconf_get_value (nnsconf_type_value type)
+{
+  int i;
+  gchar *ret;
+
+  switch (type) {
+    case NNSCONF_VAL_TF_MEM_OPTMZ:
+      for (i = 0; i < CONF_SOURCES; i++) {
+        if (conf.valueTF_MEM_OPTMZ[i]) {
+          conf.retTF_MEM_OPTMZ = g_malloc_n (1, sizeof (gchar *));
+          if (!g_ascii_strncasecmp ("1", conf.valueTF_MEM_OPTMZ[i], 1) ||
+              !g_ascii_strncasecmp ("T", conf.valueTF_MEM_OPTMZ[i], 1) ||
```

Comment:
remove this line, g_ascii_strncasecmp() compares case-insensitive str

Reply:
it's applied :)